config.js

```diff
@@ -6,28 +6,17 @@ var LIB_DEST_PATH = config.LIB_DEST_PATH;
 
 var localConfig = extend(true, {
     bowerConfig: {
-        // Bower configuration for which files to get, and where to put them.
-        // [Source, excluding bower_components]: [Destination].
-        // 'isotope/dist/isotope.pkgd.js': config.LIB_DEST_PATH,
+        'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
```

Comment:
Rewrite middleware get used in https://github.com/mozilla/marketplace-gulp/blob/503b948a60878a021723213a6e829e04f124d8d4/index.js#L337-L340 by the way.